tests/selection.py

```diff
@@ -99,20 +125,28 @@ def test_others_orthogonal(self):
 
 
 class SampleFPSTest(unittest.TestCase, PCovSelectionTest):
-    def setup(self):
+    def setUp(self):
         self.model = SampleFPS
 
+    def test_supplied_indices(self):
+        """
+        This test checks FPS will accept pre-defined indices.
+        """
+        X, Y = load_boston(return_X_y=True)
+        model = self.model(X=X, mixing=0.5, Y=Y, idxs=[0, 1])
+        model.select(n=2)
```

Comment:
Could we also check which indexes are selected? As long as we pass an initial value, it should be deterministic.